styles/prosilver/template/event/viewtopic_body_post_buttons_list_before.html

```diff
@@ -0,0 +1 @@
+<!-- DEFINE $SHOW_POST_BUTTONS = (postrow.POST_NUMBER != 1) -->
```

Comment:
I'm not sure what you mean? The idea is to not show any buttons in the first post, because the first post is the user bot.

We are inside a postrow loop when defining this, so we want to override it for the first post and force it set to false.